lib/main.coffee

```diff
@@ -18,11 +20,16 @@ module.exports = MarkdownImageAssistant =
             description: "When dragging and dropping files, whether to perserve original file names when copying over into the image directory"
             type: 'boolean'
             default: false
+        preserveFileNameInAssetsFolder:
+            title: "Use the Markdown filename when creating the assets folder"
```

Comment:
Maybe say, "Create per-file asset directories."?

Also, maybe include a description why this might be helpful: "This can help organize assets and prevent name collisions."